src/redux/actions/exposedActions.js

```diff
@@ -72,3 +73,11 @@ export const setActiveLeftPanel = dataElement => (dispatch, getState) => {
 export const setSortNotesBy = sortNotesBy => ({ type: 'SET_SORT_NOTES_BY', payload: { sortNotesBy } });
 export const updateTool = (toolName, properties) => ({ type: 'UPDATE_TOOL', payload: { toolName, properties } });
 export const setCustomPanel = newPanel => ({ type: 'SET_CUSTOM_PANEL', payload: { newPanel } });
+export const setPageLabels = pageLabels => dispatch => {
+  if (pageLabels.length !== core.getTotalPages()) {
+    console.warn('Your custom page labels don not have the same length as the number of pages in this document. This may leads to bugs in some components');
```

Comment:
I think we can just say "Number of page labels do not match with the total pages." and return

Reply:
Yeah this makes more sense, thanks!